### DISCUSSION TITLE
An item to combine all the resource scrolls and other books/documents

### DISCUSSION THREAD

**Discussion by Rutae** [ldtteam/minecolonies#9772] [Ideas]
It would be cool to see an item like the Akashic Tome or Eccentric Tome that you could use to combine all the quest books and resource scrolls into one item that you could scroll through or switch through maybe with a hotkey. If you have multiple builders the resource scrolls really start to add up lol

**Comment by Aubhisamme** [comment]
I know it's an old post. But I was searching for a functionality like this. So I support this idea even 2 years later and hope the dev see this :p
